telegram/_inline/inputtextmessagecontent.py

```diff
@@ -62,29 +79,63 @@ class InputTextMessageContent(InputMessageContent):
 
                 * |tupleclassattrs|
                 * |alwaystuple|
-        disable_web_page_preview (:obj:`bool`): Optional. Disables link previews for links in the
-            sent message.
+        link_preview_options (:obj:`LinkPreviewOptions`): Optional. Link preview generation
+            options for the message. Mutually exclusive with
+            :attr:`disable_web_page_preview`.
+
+            .. versionadded:: NEXT.VERSION
 
     """
 
-    __slots__ = ("disable_web_page_preview", "parse_mode", "entities", "message_text")
+    __slots__ = ("parse_mode", "entities", "message_text", "link_preview_options")
 
     def __init__(
         self,
         message_text: str,
         parse_mode: ODVInput[str] = DEFAULT_NONE,
         disable_web_page_preview: ODVInput[bool] = DEFAULT_NONE,
         entities: Optional[Sequence[MessageEntity]] = None,
+        link_preview_options: ODVInput["LinkPreviewOptions"] = DEFAULT_NONE,
         *,
         api_kwargs: Optional[JSONDict] = None,
     ):
         super().__init__(api_kwargs=api_kwargs)
+        if (
+            disable_web_page_preview is not DEFAULT_NONE
+            and link_preview_options is not DEFAULT_NONE
+        ):
+            raise ValueError(
+                "`disable_web_page_preview` and `link_preview_options` are mutually exclusive."
+            )
+
+        # Convert to LinkPreviewOptions:
+        if not isinstance(disable_web_page_preview, DefaultValue):
+            link_preview_options = LinkPreviewOptions(is_disabled=disable_web_page_preview)
+
```

Comment:
can this be combined with a helper function also used in `Bot`?

Reply:
hmm good point, not a high priority right now let me check soon

Reply:
ah, here it is :D